packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -37,6 +40,14 @@ interface IButtonClassNames {
 
 export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState> implements IButton {
 
+  private get _splitButton(): boolean {
+    return (this.props.menuProps != null && this.props.onClick != null);
```

Comment:
This is very strange to me but when changing the `!=` to `!==`, all of the components page become unavailable, and the console prints `too much recursion`.
Does that make any sense to you @jair-cazarin?